pkgs/development/libraries/mesa/default.nix

```diff
@@ -90,124 +89,126 @@ let self = stdenv.mkDerivation {
   #  revive ./dricore-gallium.patch when it gets ported (from Ubuntu), as it saved
   #  ~35 MB in $drivers; watch https://launchpad.net/ubuntu/+source/mesa/+changelog
   patches = [
-    ./symlink-drivers.patch
     ./missing-includes.patch # dev_t needs sys/stat.h, time_t needs time.h, etc.-- fixes build w/musl
-    ./disk_cache-include-dri-driver-path-in-cache-key.patch
   ] ++ lib.optional stdenv.isDarwin ./darwin-clock-gettime.patch;
 
   outputs = [ "out" "dev" "drivers" ]
             ++ lib.optional (elem "swrast" galliumDrivers) "osmesa";
 
   # TODO: Figure out how to enable opencl without having a runtime dependency on clang
-  configureFlags = [
+  mesonFlags = [
+    # https://gitlab.freedesktop.org/search?utf8=✓&snippets=&scope=&search=get_option('datadir')&project_id=176
+    "--datadir=${placeholder "out"}/share" # "vendorjsondir=\${out}/share/glvnd/egl_vendor.d" https://gitlab.freedesktop.org/mesa/mesa/blob/master/src/egl/meson.build#L156
```

Comment:
This should not be needed. It defaults to `share` which `install_dir` appends to `prefix` and in C constants, they are already joining it with `prefix`.